```
Performing static analysis using mypy
pandas/core/internals/blocks.py:2119: error: Definition of "to_native_types" in base class "DatetimeLikeBlockMixin" is incompatible with definition in base class "Block"  [misc]
pandas/core/internals/blocks.py:2365: error: Definition of "to_native_types" in base class "DatetimeLikeBlockMixin" is incompatible with definition in base class "Block"  [misc]
Found 2 errors in 1 file (checked 1057 source files)
```

--- a/pandas/core/internals/blocks.py
+++ b/pandas/core/internals/blocks.py
@@ -2070,7 +2070,7 @@ class IntBlock(NumericBlock):
         return is_integer(element) or (is_float(element) and element.is_integer())
 
 
-class DatetimeLikeBlockMixin:
+class DatetimeLikeBlockMixin(Block):
     """Mixin class for DatetimeBlock, DatetimeTZBlock, and TimedeltaBlock."""
 
     @property
@@ -2116,7 +2116,7 @@ class DatetimeLikeBlockMixin:
         return self.make_block(result)
 
 
-class DatetimeBlock(DatetimeLikeBlockMixin, Block):
+class DatetimeBlock(DatetimeLikeBlockMixin):
     __slots__ = ()
     is_datetime = True
 

